_posts/code/rspec/2016-04-09-organizing-config.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Cleaning up your spec/rails helpers
+category: code
+description: "Organizing your RSpec configurations"
+---
+
+A common pattern when working on a Rails project is to add all RSpec configuration
+to the `spec_helper` or `rails_helper`. Overtime this will cause these files to grow
+and make them hard to maintain. A useful approach to cleaning this up is to add
+configurations for specific libraries in their own files.
+
+Adding the following line in your `spec_helper` or `rails_helper` will load all files in
+`spec/support`.
+
+~~~ ruby
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
+~~~
+
+The next step is to move/create new configs in separate files in `spec/support`.
+For example your database cleaner config can be stored in
+`spec/support/database_cleaner.rb`.
+
+~~~ ruby
+RSpec.configure do |config|
+  config.use_transactional_fixtures = false
+
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.around(:each) do |example|
+    DatabaseCleaner.cleaning do
+      example.run
+    end
+  end
+end
+~~~
```

Comment:
I feel like there should be a couple sentences at the end about why this might be beneficial, about how you will always know that everything related to database cleaner in your test suite will be in the spec/support/database_cleaner.rb file, and that if you ever want to remove it you can just rm the file